source/fuzz/fuzzer_context.h

```diff
@@ -62,6 +62,37 @@ class FuzzerContext {
     return result;
   }
 
+  // Randomly shuffles a |sequence| between |lo| and |hi| indices inclusively
+  // |lo| and |hi| must be valid indices to the |sequence|
+  template <typename T>
+  void Shuffle(std::vector<T>* sequence, size_t lo, size_t hi) const {
+    auto& array = *sequence;
+
+    if (array.empty()) {
+      return;
+    }
+
+    assert(lo <= hi && hi < array.size() && "lo and/or hi indices are invalid");
+
+    // i > lo to account for potential infinite loop when lo == 0
+    for (size_t i = hi; i > lo; --i) {
+      auto index = random_generator_->RandomUint32(
+          static_cast<uint32_t>(i - lo + 1));
+
+      if (lo + index != i) {
+        using std::swap;
+        swap(array[lo + index], array[i]);
```

Comment:
Please change "swap" to "std::swap" and get rid of the "using".  In general we avoid "using" in this project.

Reply:
No problem ;) Although, I thought it's a good practice from [this post](http://ericniebler.com/2014/10/21/customization-point-design-in-c11-and-beyond/)

Reply:
That’s interesting: by all means keep it in that case, with a short comment stating that it caters for a swap other than std::swap being used.

Reply:
Fixed in fa6532d